### ISSUE TITLE
xxhash was compiled against a different Node.js version

### ISSUE THREAD

**Issue by tangb** [balena-io/balena-cli#1419]
- **balena CLI version:** e.g. 11.2.1 (output of the `"balena version"` command)
balena-cli v11.9.3

- **Operating system version:** e.g. Windows 10, Ubuntu 18.04, macOS 10.14.5
windows10

- **32/64 bit OS and processor:** e.g. 32-bit Windows on 64-bit Intel processor
64bits

- **Install method:** npm or zip or executable installer
zip archive available https://github.com/balena-io/balena-cli/releases/tag/v11.9.3

- **If npm install, Node.js and npm version:** e.g. Node v8.16.0 and npm v6.4.1
npm 6.11.2
node v10.16.3
---

Hello

I'm trying to run balena-cli command line under windows 10  
`balena.exe local flash "C:\temp\raspbian.zip" --drive "\\.\PHYSICALDRIVE1" --yes`

and I got the following error:
```
The module '\\?\c:\Users\tanguy.bonneau\AppData\Roaming\CleepDesktop\balena-cli\xxhash\build\Release\hash.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 57. This version of Node.js requires
NODE_MODULE_VERSION 64. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`).

Additional information may be available by setting a DEBUG=1 environment
variable: "set DEBUG=1" on a Windows command prompt, or "export DEBUG=1"
on Linux or macOS.

If you need help, don't hesitate in contacting our support forums at
https://forums.balena.io

For bug reports or feature requests, have a look at the GitHub issues or
create a new one at: https://github.com/balena-io/balena-cli/issues/
```

I understand the error, but I don't figure out how to find a workaround while the binary is delivered in the release archive (no way to rebuild it).

Maybe you can help me ?
I saw a new release of xxhash has been released few days ago after 3 years of inactivity.

Thank you very much for your answer ;-)

**Comment by pdcastro** [comment]
@tangb, thank you for reporting this issue! I have reproduced it and I believe it is a bug in our CI (continuous integration) build scripts, which I will investigate further and fix. Meanwhile, I have tested the CLI's "executable installer" for Windows and it seems that it is not affected by this issue, so you could use it instead of the standalone zip package:
* [balena-cli-v11.9.3-windows-x64-installer.exe](https://github.com/balena-io/balena-cli/releases/download/v11.9.3/balena-cli-v11.9.3-windows-x64-installer.exe) or [later](https://github.com/balena-io/balena-cli/releases)
* https://github.com/balena-io/balena-cli/blob/master/INSTALL.md#executable-installer

**Comment by tangb** [comment]
Great
Thank you very much ;-)

**Comment by pdcastro** [comment]
* Issue fixed in CLI release 11.9.4 and later ([releases](https://github.com/balena-io/balena-cli/releases), [changelog](https://github.com/balena-io/balena-cli/blob/master/CHANGELOG.md)).
* I've gathered that this issue affected standalone zip packages for Windows only, between CLI releases 11.4.0 and 11.9.3 (inclusive range).
* The fix pull requests also added automated testing of the standalone zip package to prevent any future regression.

Thanks again for reporting this issue.
